tools/testbench/common_test.c

```diff
@@ -100,7 +100,7 @@ int tb_pipeline_params(struct ipc *ipc, struct pipeline *p,
 {
 	struct ipc_comp_dev *pcm_dev;
 	struct comp_dev *cd;
-	struct sof_ipc_pcm_params params;
+	struct sof_ipc_pcm_params params = {0};
```

Comment:
This looks like the initialization of some field(s) is missing below and this is merely hiding the issue.

Reply:
ditto as above.